fabric-content-registries-v1/src/main/java/net/fabricmc/fabric/mixin/content/registries/MixinPlayerEntity.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.content.registries;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.entity.player.PlayerInventory;
+import net.minecraft.item.ItemStack;
+
+import net.fabricmc.fabric.api.content.registry.v1.item.UseTickable;
+
+@Mixin(PlayerEntity.class)
+public class MixinPlayerEntity {
+	@Shadow
+	public PlayerInventory inventory;
+
+	@Shadow
+	private ItemStack field_8356;
```

Comment:
would be nice to get yarn mappings for this

Reply:
probably fixed in the big pr

Reply:
it's `itemInUse`

Reply:
fix thank

Reply:
fixed in my yarrn pr